release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1930,6 +1935,45 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, {product-title} upgrades for Ansible caused an error as the IPsec configuration was not idempotent. With this release, the issue is resolved. Now, all IPsec configurations for OpenShift Ansible playbooks are idempotent. (link:https://issues.redhat.com/browse/OCPBUGS-30802[*OCPBUGS-30802*])
+
+* Previously, using network types with dual-stack networking other than Open vSwitch-based software-defined networking (SDN) or Open Virtual Network (OVN) caused a validation error. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30232[*OCPBUGS-30232*])
+
+* Previously, a closed IPv6 port range for `nodePort` services in user-provisioned-infrastructure installations on {rh-openstack} caused traffic through certain node ports to be blocked. With this release, appropriate security group rules have been added to the `security-group.yaml` playbook, resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-30154[*OCPBUGS-30154*])
+
+* Previously, manifests that were generated by using the command ‘openshift-install agent create cluster-manifests’ command were not directly applied to an {product-titl} cluster because the manifests did not include type data. With this release, type data has been added to the manifests. Administrators can now apply the manifests to initiate a Zero Touch Provisioning (ZTP) installation that uses the same settings as the Agent-based installation. (link:https://issues.redhat.com/browse/OCPBUGS-29968[*OCPBUGS-29968*])
+
+* Previously, a file required for the `aarch64` architecture was renamed by mistake while generating the `aarch64` agent ISO. With this release, the specified file does not get renamed.(link:https://issues.redhat.com/browse/OCPBUGS-28827[*OCPBUGS-28827*])
+
+* Previously, when installing a cluster on {vmw-first}, the installation would fail if an ESXi host was in maintenance mode due to the installation program failing to retrieve version information from the host. With this release, the installation program does not attempt to retrieve version information from ESXi hosts that are in maintenance mode, allowing the installation to proceed. (link:https://issues.redhat.com/browse/OCPBUGS-27848[*OCPBUGS-27848*])
+
+* Previously, the {ibm-cloud-name} Terraform Plugin incorrectly prevented the use of non-private service endpoints during cluster installation. With this release, the {ibm-cloud-name} Terraform Plugin supports non-private service endpoints during installation. (link:https://issues.redhat.com/browse/OCPBUGS-24473[*OCPBUGS-24473*])
+
+* Previously, installing a cluster on {vmw-first} required specifying the full path to the data store. With this release, the installation program accepts full paths and relative paths for the data store. (link:https://issues.redhat.com/browse/OCPBUGS-22410[*OCPBUGS-22410*])
```

Comment:
(I think they have it as one word, if that's wrong disregard)
```suggestion
* Previously, installing a cluster on {vmw-first} required specifying the full path to the datastore. With this release, the installation program accepts full paths and relative paths for the datastore. (link:https://issues.redhat.com/browse/OCPBUGS-22410[*OCPBUGS-22410*])
```

Reply:
:heavy_check_mark: